gammapy/cube/core.py

```diff
@@ -324,6 +320,10 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos):
```

Comment:
Wouldn't it be more natural to give an energy (e.g. center energy) here instead of of `slicepos`?